### ISSUE TITLE
Accessibility (WCAG level AA) Links in the site's link bar have insufficient contrast

### ISSUE THREAD

**Issue by claycompton** [NuGet/NuGetGallery#1167]
1.  Download Paciello Group's Contrast Analyser from http://www.paciellogroup.com/resources/contrastAnalyser.  This tool measures foreground-to-background contrast ratios using the same algorithm defined by the W3C's Web Content Accessibility Guidelines.
2.  Navigate to http://nuget.org/.
3.  Compare the contrast ratio of the links in the link bar in the site footer against their backgrounds.

Result:  It's 4.46:1.  
Expected:  This is below the threshold of 4.5:1 given in WCAG.  We should be able to fix this easily by slightly adjusting our color palette.

**Comment by johnataylor** [comment]
We have a new gallery design.
